### ISSUE TITLE
KPL Producer 0.15.7 throwing Could not copy native binaries to temp directory

### ISSUE THREAD

**Issue by Ber1984** [awslabs/amazon-kinesis-producer#516]
Hey. 
I'm trying to use KPL 0.15.7 latest version and i copied the code kinesis-producer-library-examples-master and run the C_AsyncProducer example locally but i'm getting the next error 
```
Could not copy native binaries to temp directory 
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.InputStream.read(byte[])" because "input" is null
```
The error is exactly here `final KinesisProducer kinesis = new KinesisProducer(config);`

and if i downgrade to 0.13.1 I'm getting this error when i call `addUserRecord()` method
`The child process has been shutdown and can no longer accept messages.`

Any help?

**Comment by kothapet** [comment]
Same here.. Looking at history, looks like this repeats every few releases. Please fix it.

**Comment by erickjhorman** [comment]
> Same here.. Looking at history, looks like this repeats every few releases in windows. Please fix it.

Yeah, I had to use 0.13.1 to fix the problem about addUserRecord()
Looking forward for a solution to use the latest version.
